setup.py

```diff
@@ -1,25 +1,36 @@
+import sys
 from setuptools import setup
 
+# Extra dependecies to run tests
+tests_requirements = [
+    "pytest>=4.6.0",
+    "timeout-decorator",
+    "funcy>=1.14",
```

Comment:
```suggestion
    "funcy>=1.14,<2",
```
If I'll break backwards compatibility. Same for `pytest` I guess.